generator/internal/genclient/parser/protobuf/mixin.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package protobuf
+
+import (
+	"cloud.google.com/go/iam/apiv1/iampb"
+	"cloud.google.com/go/longrunning/autogen/longrunningpb"
+	"google.golang.org/genproto/googleapis/api/serviceconfig"
+	"google.golang.org/genproto/googleapis/cloud/location"
+	"google.golang.org/protobuf/reflect/protodesc"
+	"google.golang.org/protobuf/types/descriptorpb"
+)
+
+const (
+	locationPackage    = "google.cloud.location"
+	locationService    = locationPackage + ".Locations"
+	iamPackage         = "google.iam.v1"
+	iamService         = iamPackage + ".IAMPolicy"
+	longrunningPackage = "google.longrunning"
+	longrunningService = longrunningPackage + ".Operations"
+)
+
+var enabledMixins = map[string]bool{
+	locationPackage:    false,
+	iamPackage:         false,
+	longrunningService: false,
+}
```

Comment:
This should be a property of the API.

Eventually we may want to run multiple Codec / parsers in the same sidekick instance (e.g. to regenerate the world using multiple goroutines).